projects/angular/src/data/datagrid/render/main-renderer.ts

```diff
@@ -59,6 +60,7 @@ export class DatagridMainRenderer implements AfterContentInit, AfterViewInit, Af
   private _heightSet = false;
   private shouldStabilizeColumns = true;
   private subscriptions: Subscription[] = [];
+  private isVisible = false;
```

Comment:
Wait, you don't need this property. You just return the value.

Reply:
Maybe rename `setVisibility` to `checkAndUpdateVisibility` and return the value.